drivers/wifi/nrf700x/zephyr/src/qspi/src/qspi_if.c

```diff
@@ -122,6 +115,10 @@ static inline int qspi_freq_to_sckfreq(int freq)
 #endif
 }
 
+static inline int qspi_dts_freq_to_sckfreq(void)
+{
+	return qspi_freq_to_sckfreq(INST_0_SCK_FREQUENCY);
```

Comment:
The function qspi_freq_to_sckfreq() processes an input in MHz unit. Nevertheless, INST_0_SCK_FREQUENCY is specified in Hz. Therefore, a 24MHz in DTS will become 6MHz in this function...